packages/components/message/src/message-method.ts

```diff
@@ -60,26 +60,26 @@ const normalizeOptions = (params?: MessageParams) => {
   return normalized as MessageParamsNormalized
 }
 
+const getNextOffset = (offset: number) => offset + 48 + 16
```

Comment:
Where does 48 come from and should we annotate it?

Reply:
The height of Message is `48`, `16` is the gap height of each messages.

Reply:
Updated.